compiler/rustc_codegen_rmc/src/codegen/rvalue.rs

```diff
@@ -721,7 +721,17 @@ impl<'tcx> GotocCtx<'tcx> {
 
         // Lookup in the symbol table using the full symbol table name/key
         let fn_name = self.symbol_name(instance);
-        if let Some(fn_symbol) = self.symbol_table.lookup(&fn_name) {
+
+        if let Some(fn_symbol) = self.symbol_table.clone().lookup(&fn_name) {
```

Comment:
This clone is going to be super expensive. Please clone after the lookup!

Reply:
Totally just a mistake, thanks!